.travis.yml

```diff
@@ -6,30 +6,32 @@ services:
 dist: trusty
 
 go:
```

Comment:
Is this really required? We build in-docker anyway.

Reply:
I'm not a Travis CI expert, but omitting it will result in a Travis error :-/

Reply:
I'm used to CircleCI, there you can pick a Docker image that's the context for the entire build. I'm not sure if that's possible on Travis, but that would be extra reason to use [the concept of a "build" docker image](https://github.com/h2non/imaginary/pull/229).

Reply:
Would there be a problem exploring migrating to CircleCI? I have a lot of experience using them too..

Reply:
We'd lose the "matrix" functionality, although I find it severely lacking for my train of thought. Plus Travis is a bit of an industry standard for OSS. I'm not sure what we'd lose specifically. I'd like @h2non feedback on this, as it's "quite a change".

Reply:
We could always tinker in parallel and then build a case towards why a migration would improve the current state? Matrix build visibility could be done via checks here at github (if we're attempting to cover the UI part - I believe workflows would give us the variation per-job).